values.yaml

```diff
@@ -328,10 +328,24 @@ syncCatalog:
   # to all namespaces.
   k8sSourceNamespace: null
 
-  # consulPrefix is the service prefix which preprends itself
+  # addK8SNamespaceSuffix appends Kubernetes namespace suffix to
+  # each service name synced to Consul, separated by a dash.
+  # For example, for a service 'foo' in the default namespace,
+  # the sync process will create a Consul service named 'foo-default'.
+  # Set this flag to true to avoid registering services with the same name
+  # but in different namespaces as instances for the same Consul service.
+  # Namespace suffix is not added if 'annotationServiceName' is provided.
+  # NOTE: Setting this property to 'true' will result in deregistering of existing
```

Comment:
This is only a concern when someone is updating an existing Consul installation. This comment reads to me like it might happen if someone sets it to `true` the first time they install. 

Also, given that we're now defaulting this to `true`, the wording is a little odd.

Reply:
True. I removed that note entirely. This comment will be more appropriate in the Changelog.